addons/koji/common/src/main/java/org/commonjava/indy/koji/conf/IndyKojiConfig.java

```diff
@@ -449,7 +463,7 @@ public void parameter( final String name, final String value )
                         targetGroups = new LinkedHashMap<>();
                     }
 
-                    String source = name.substring( "target.".length(), name.length() );
+                    String source = name.substring( TARGET_KEY_PREFIX.length(), name.length() );
```

Comment:
I think my comments in the design weren't very clear on this point, but I was hoping we could still specify that proxied binary (import) builds could be added to different targets than real source builds that we proxy from Koji. From this, it looks like we're only using a single target...is that right?